woocommerce/woocommerce-gutenberg-products-block
# Order confirmation block styling

## What

This PR restores https://github.com/woocommerce/woocommerce-blocks/pull/10606 and applies styles (padding, alignment, colors) to finalise the order confirmation page.

Fixes #10763

## Why

n/a

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to Appearance > Editor > Templates, select Order Confirmation
2. If you've made edits, click the three dots and "reset customisations"
3. Hover over the order confirmation placeholder and "convert to blocks"
4. You should see the correct layout now including summary, status, details, downloads, address sections
5. Confirm the design matches the screenshot below
6. Try applying some changes such as background and color changes. One to test is the summary block; give it a background color and confirm it gets automatic padding of 16px once it refreshes
7. Save changes
8. Place an order and confirm the frontend matches the preview state


## Screenshots or screencast

![screenshot_2023-08-30_at_16 16 24_720](https://github.com/woocommerce/woocommerce-blocks/assets/90977/be6e49a8-d0cf-40a4-abe3-681320dcf85f)

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog
 
n/a
